CapuaEnvironment/ExecutionUnit/ExecutionUnit.py

```diff
@@ -24,22 +24,7 @@
 from CapuaEnvironment.IntructionFetchUnit.InstructionFetchUnit import InstructionFetchUnit
 from CapuaEnvironment.IntructionFetchUnit.FormDescription import formDescription
 from CapuaEnvironment.IOComponent.MemoryIOController import MemoryIOController
-from Configuration.Configuration import REGISTER_A, \
-                                        REGISTER_B, \
-                                        REGISTER_C, \
-                                        REGISTER_D, \
-                                        REGISTER_E, \
-                                        REGISTER_F, \
-                                        REGISTER_G, \
-                                        REGISTER_A2, \
-                                        REGISTER_B2, \
-                                        REGISTER_C2, \
-                                        REGISTER_D2, \
-                                        REGISTER_E2, \
-                                        REGISTER_F2, \
-                                        REGISTER_G2, \
-                                        REGISTER_S, \
-                                        REGISTER_S2, \
+from Configuration.Configuration import REGISTERS, \
```

Comment:
I know this feels like a good idea but this PR is now much more extensive than "just the assembler". This change also force a dictionary lookup every time a register is accessed. I will have to profile this for performance before I allow it in or not.

Reply:
Honestly this was mainly to shorten the code and make it easier to navigate the assembler. If it creates too much of a conflict with the executionUnit I can revert it